src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -25,7 +27,8 @@ const (
 )
 
 var (
-	log = golog.LoggerFor("flashlight.analytics")
+	log  = golog.LoggerFor("flashlight.analytics")
+	elog = errlog.ErrorLoggerFor("flashlight.analytics")
```

Comment:
Making this differentiation is so some errors are reported using this mechanism and others using the traditional GoLog? I would propose using a common interface for both, unless there is a strong reason to keep these separate.